packages/back-end/src/controllers/features.ts

```diff
@@ -1211,6 +1331,40 @@ export async function postFeatureMoveRule(
     version: revision.version,
   });
 }
+export async function getDraftandReviewRevisions(
+  req: AuthRequest,
+  res: Response
+) {
+  const context = getContextFromReq(req);
+  const revisions = await getRevisionsByStatus(context, [
+    "draft",
+    "approved",
+    "changes-requested",
+    "pending-review",
+  ]);
+  const featuresAndRevisions: {
+    revision: FeatureRevisionInterface;
+    feature: FeatureInterface | null;
+  }[] = [];
+
+  for (const r of revisions) {
+    if (r) {
+      const feature = await getFeature(context, r.featureId);
+      // we need to filter out features that are created after the revision
+      // eg someone deletes a feature with revisions and creates a new one with the same id
+      if (feature && feature.dateCreated.getTime() < r.dateCreated.getTime()) {
```

Comment:
Use `<=` instead of just `<`.  Otherwise, I think there's a chance the very first revision might be missing since that gets created at the same time as the feature.